src/index.html

```diff
@@ -4,14 +4,166 @@
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+          >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </label>
+      </fieldset>
+    
+      <fieldset class="form-group registration_group">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email" 
+            name="e-mail" 
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="8" 
+            maxlength="32" 
+            required
+          >
+        </label>       
+      </fieldset>
+
+      <fieldset class="form-group user-facts_group">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?         
+          <input 
+            type="radio"
+            name="loveCat"
+            value="yes"
+          >
+          Yes
+          <input 
+            type="radio" 
+            name="loveCat" 
+            value="no"
+          >
+          No
+        </label>
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>       
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time_to_bed"
+          >
+        </label>       
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select 
+            name="car"
+            required
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="AUDI">AUDI</option>
+            <option value="BYD">BYD</option>
+            <option value="Choose cars"
+              selected
+              disabled>
+            </option>
+          </select>
+        </label>       
+        <label class="form-field">  
+          How do you rate our work?
+          <input
+            type="range"
+            name="rating"
+            min="1"
+            max="10"
+          >
+        </label>       
+      </fieldset>
+      
+      <fieldset class="form-group additional-info_group">
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments
+          <textarea
+            name="Comment"
+            cols="30"
+            rows="2">
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="Comment"
            cols="30"
            rows="2"></textarea>
```
it has space now
<img width="260" alt="image" src="https://user-images.githubusercontent.com/102411130/216555482-0ab6475d-ca87-472c-93cb-de5ca0fa66f1.png">

Reply:
done